### DISCUSSION TITLE
#37 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by bobjiang** [tog-ETH-er/togETHer#12] [General]
Agenda:

1. 下一届轮值主席提名
2. Community Updates
3. Fireside discussion

会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/s/LxNajKWJw4RV

会议 ID：891 5098 6775

**Comment by bambalados** [comment]
1) 在CTN Twitter Space 做了一个社区Talk
2) 准备下一周关于the merge的CTN活动。正好想问问有没有togETHer的小伙伴愿意给一个The Merge的overview keynote.

**Comment by winkybupt** [comment]
# 原语里弄
- 四月共度活动第一本书共读完结，并举办了读书会；第二本书共读开始；
- 支持 CTN 举办 the merge 的活动

**Comment by Riely007** [comment]
### 以太坊翻译小组

- 官网翻译 review 中
- 周边设计 80%

**Comment by 0x-stan** [comment]
# Dapp-Learning

近两周工作进展

## 一周年红包活动

向贡献者发放随机红包奖励

## repo update

13个PR #689 ～ #702

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了3场分享会

- Thorchain项目解读
  - by Koi
  - https://youtu.be/URQmzmA-KAw
- ethereum p2p code review
  - by hhh
  - https://youtu.be/8pKdVdeBw00
  - https://youtu.be/-flrVVcZshU
- yugalabs系NFT介绍&代码解读
  - by arc
  - https://youtu.be/mAttey5Xrhc
  - https://youtu.be/n-g4ek-Pxeo


## 翻译小组

- 正式发布第 12、13集
  - https://mp.weixin.qq.com/s/MuLVY1-qlzGeLdNLe7d0Fg
  - https://mp.weixin.qq.com/s/kXj9tQk2Y-NVd5RMobKJsQ
- 翻译小组内部经验交流会议

**Comment by bobjiang** [comment]
- GitcoinDAO - user support 
- Gitcoin claim matching funds
- ENS DAO, look for ecosystem projects
- zkSync video https://www.youtube.com/playlist?list=PLnP6dU8KobC9mLQprlOu1ucnDe48RXUu-

**Comment by stephanieeee-ho** [comment]
## ECN
- 常规的翻译和周报输出
- 策划 The Merge 主题活动

**Comment by Emilyxiannv** [comment]
### DAOSquare
项目研究分享：
NFT借贷协议 BendDAO（https://mirror.xyz/daojoker.eth/AHcEe4crwupu-At_MUrZoJvASUJK6VDeOUsXl4MNcHk）
分享人： @Crypto_joker from DAOSquare community
分享讨论视频：
https://www.youtube.com/watch?v=oTH4DoYnv-Y

**Comment by Jackieyewang** [comment]
登月工坊黑客松和链坊科技黑客松宣传协助

**Comment by luckyyang** [comment]
### Rebase
- 欢迎大家去这里[发布 event](https://github.com/rebase-network/event-pool/issues)
- Web3 Daily 更新 https://rebase.network/posts/category/web3daily
